grpc/diatheke.proto

```diff
@@ -281,10 +306,40 @@ message PushTextRequest {
   string text = 2;
 }
 
+// Request to change the current story of a session.
+message StoryRequest {
+  // ID of the session that will have its story changed.
+  string session_id = 1;
+
+  // ID of the story to switch to. This ID is defined by the model
+  // used to create the session.
+  string story_id = 2;
+
+  // Parameters to set in session memory before executing the specified
+  // story. Some stories in the model may make assumptions about which
+  // parameters have already been defined, so it is important to be
+  // familiar with the model requirements for any given story.
+  map<string, string> parameters = 3;
```

Comment:
Are there use cases for sending parameter changes _without_ changing the story_id?